types/types.go

```diff
@@ -252,6 +252,8 @@ type Info struct {
 	ClusterStore       string
 	ClusterAdvertise   string
 	SecurityOptions    []string
+	Runtimes           map[string]Runtime
```

Comment:
We do:

```
# docker info
...
Runtimes:
 oci: /usr/local/bin/docker-runc (--debug)
 default: docker-runc
 foo: /usr/local/bin/foo (--debug)
Default Runtime: foo
...
```

It's part of the Runtime struct, we also display the runtime arguments if any were defined